packages/react/src/grid/GridCell.tsx

```diff
@@ -0,0 +1,29 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import clsx from 'clsx'
+import { ReactNode } from 'react'
+import { PageGridColumnNumber } from './PageGrid'
+
+export type GridCellProps = {
+  children: ReactNode
+  gridColumns?: [PageGridColumnNumber, PageGridColumnNumber]
+}
+
+export const gridCellClassNames = (gridColumns?: [PageGridColumnNumber, PageGridColumnNumber]) => {
+  if (!gridColumns) {
+    return undefined
+  }
+
+  const [start, end] = gridColumns
+
+  return clsx(start && `amsterdam-grid-start-${start}`, end && `amsterdam-grid-end-${end}`)
+}
+
+export const GridCell = ({ children, gridColumns }: GridCellProps) => (
```

Comment:
Decide whether we need this component if many (if not all) other components will include `GridColumnProps` themselves.

Reply:
Let’s start with just `Paragraph` extending `GridCellProps` and add more later.